code/modules/fish/fish_tank.dm

```diff
@@ -800,16 +800,24 @@
 		multiplier = initial(multiplier) + (C.rating*0.1) //1 to 1.2
 
 /obj/machinery/power/conduction_plate/process()
+	if(!anchored)
+		return
+	var/power = 0
 	if(check_tank())
-		var/power = 0
 		for(var/fish in attached_tank.fish_list)
 			if(fish == "electric eel")
 				power += ARBITRARILY_LARGE_NUMBER * multiplier //10000
```

Comment:
Not related to this PR but what the fuck is this

Reply:
A classic define

Reply:
Classic misuse of a define, that is